src/pages/task/ActionButtons.tsx

```diff
@@ -250,6 +256,18 @@ export const ActionButtons: React.FC<Props> = ({
   return (
     <>
       <PageButtonRow>
+        <LeafyGreenButton
```

Comment:
Shouldn't need to import a separate button component component/button should implement all the features of a lg button.